spec/javascripts/support/jasmine_helper.rb

```diff
@@ -1,4 +1,12 @@
-# Prevent PhantomJS auto install, uses PhantomJS already on your path.
+require "jasmine/runners/selenium"
+require "webdrivers/chromedriver"
+
 Jasmine.configure do |config|
   config.prevent_phantom_js_auto_install = true
+  config.runner = lambda { |formatter, jasmine_server_url|
+    options = Selenium::WebDriver::Chrome::Options.new
+    options.headless!
+    webdriver = Selenium::WebDriver.for(:chrome, options: options)
```

Comment:
I think a change similar to this may also need to be made to allow Chromedriver to run: https://github.com/alphagov/travel-advice-publisher/blame/0d772912137b067bc0e3615c504025a66f1917ec/spec/support/webmock.rb

Reply:
There's part of that line in test_helper.rb 

```
WebMock.disable_net_connect!(allow_localhost: true)
```

I've tried adding `allow: ["chromedriver.storage.googleapis.com"]` and haven't noticed a difference in the way the tests run

Reply:
This looks to be something that was a local issue only, so clearly that line in test_helper.rb is doing what it is supposed to and no changes are needed after all.